docs/topics/exporters.rst

```diff
@@ -311,7 +311,7 @@ CsvItemExporter
 
    The additional keyword arguments of this ``__init__`` method are passed to the
    :class:`BaseItemExporter` ``__init__`` method, and the leftover arguments to the
-   `csv.writer`_ ``__init__`` method, so you can use any ``csv.writer`` ``__init__`` method
+   :func:`csv.writer` ``__init__`` method, so you can use any ``csv.writer`` ``__init__`` method
```

Comment:
The change makes it obvious that the documentation is wrong. It looks like we should replace “`__init__` method” with “function” in the 2 instances in this line.

Reply:
I have changed it to
```rst
:func:`csv.writer` function, so you can use any :func:`csv.writer` function
   argument to customize this exporter.
```